Tuist/ProjectDescriptionHelpers/Target+Initializing.swift

```diff
@@ -23,27 +25,31 @@ extension Target {
     }
     
     public static func testsTarget(name: String, bundleId: String = "co.nimblehq") -> Target {
+        let targetName = "\(name)Tests"
         return Target(
-            name: "\(name)Tests",
+            name: targetName,
             platform: .iOS,
             product: .unitTests,
-            bundleId: "\(bundleId).\(name)Tests",
-            infoPlist: .default,
-            sources: ["\(name)Tests/**"],
+            bundleId: "\(bundleId).\(targetName)",
+            infoPlist: "\(targetName)/\(plistsPath)/Info.plist",
+            sources: ["\(targetName)**"],
```

Comment:
Sorry, I missed this one.
```suggestion
            sources: ["\(targetName)/**"],
```